packages/jest-message-util/src/index.ts

```diff
@@ -355,3 +350,10 @@ export const separateMessageFromStack = (content: string) => {
   const stack = messageMatch[2];
   return {message, stack};
 };
+
+export const formatFullTitle = (result: AssertionResult) =>
+  result.ancestorTitles.join(ANCESTRY_SEPARATOR) +
+  (result.ancestorTitles.length ? ANCESTRY_SEPARATOR : '') +
+  result.title;
+
+export {TITLE_INDENT, TITLE_BULLET};
```

Comment:
please add `export` in front the the declaration instead of a separate `export` statement down here 🙂